back/engines/commercial/bulk_import_ideas/spec/models/idea_import_spec.rb

```diff
@@ -50,8 +50,19 @@
           expect { user.reload }.to raise_error(ActiveRecord::RecordNotFound)
         end
 
+        it 'does not delete a user when the user has a password' do
```

Comment:
Should these tests be in `Patches::Idea` specs? When I tried to find the code that removes a user, I went to `bulk_import_ideas/idea_import.rb`, but it wasn't there to my confusion.